src/ceph-daemon/ceph-daemon

```diff
@@ -1826,12 +1833,15 @@ def _get_parser():
         help='cluster FSID')
     parser_bootstrap.add_argument(
         '--output-keyring',
+        default='ceph.client.admin.keyring',
         help='location to write keyring file with new cluster admin and mon keys')
     parser_bootstrap.add_argument(
         '--output-config',
+        default='ceph.conf',
```

Comment:
Perhaps we set the default to `/etc/ceph/ceph.conf` etc. for the cases when `--output-config` and friends are not specified?

Reply:
I was wondering about this. 

1. /etc/ceph won't exist if the packages aren't installed, which they usually won't be (e.g., if the user is following the install instructions).
2. If we rely on this to set up their 'admin' CLI host, then they only have 1--where the bootstrapped, and still need to manually do this on other host(s)
3. They can get the same effect if the cd /etc/ceph and then run bootstrap from there

More generally, ceph-deploy had the 'admin' command that would install the ceph.conf and client.admin keyring on a remote host.  We don't really have an equivalent currently.  But.. I'm not sure if we should.  It might also/instead make sense to have ceph-daemon shell look for /etc/ceph/* by default and, if present bind those into the container...

Reply:
For now, this PR feels like a step in the right direction

Reply:
> More generally, ceph-deploy had the 'admin' command that would install the ceph.conf and client.admin keyring on a remote host. We don't really have an equivalent currently. But.. I'm not sure if we should. It might also/instead make sense to have ceph-daemon shell look for /etc/ceph/* by default and, if present bind those into the container...

hmm yeah ..  it doesn't feel like ceph-deamon should interact with remote hosts.



> For now, this PR feels like a step in the right direction

yep, agree